gorillaz.go

```diff
@@ -41,6 +44,9 @@ type Gaz struct {
 	serviceAddress        string // optional address of the service that will be used for service discovery
 	streamConsumers       *streamConsumerRegistry
 	streamEndpointOptions []StreamEndpointConfigOpt
+	httpListener          net.Listener
+	httpSrv               *http.Server
+	prometheusRegistry    *prometheus.Registry
```

Comment:
Not sure if it matters, but there is even an interface for the Registry:
https://github.com/prometheus/client_golang/blob/master/prometheus/registry.go#L95

Reply:
thanks, I did not see that, I will use the interface.

Reply:
well in fact I need to keep the Registry because it implements Registerer and Gatherer